css/modals/virtual-background/_virtual-background.scss

```diff
@@ -85,10 +85,26 @@
 
     @media (min-width: 432px) and (max-width: 632px) {
         font-size: 1.5vw;
+        .desktop-share, .virtual-background-none, .thumbnail, .blur, .slight-blur{
```

Comment:
Same here. Also, since these are separate selectors, could we have them on new lines for readability's sakes?